### PR TITLE
Feature/439 crowdloan fund detail page

### LINKED ISSUE #439: Crowdloan fund details screen
> As a user I'd like to have more information about a fund 

When the user click on one of the fund in the crowdloan list it opens the following with the following information:
- Project name
- Index
- Depositor 
- Ending time
- Leases
- Amount raised
- Count
- Status

* Wireframe: 

* When the user clicks on homepage it opens that project website

### PR DESCRIPTION
**Pull request by ahoseinian**
- Add detail page for Crowd loan funds
- The navigator will be fixed in #444

fixes #439

### CODE REVIEW

File: src/api/hooks/useFunds.ts [hunk 1]
```diff
@@ -18,39 +19,57 @@ export const CROWD_PREFIX = stringToU8a('modlpy/cfund');
...
     },
     {enabled: !!bestNumber},
   );
 }
 
+export function useCrowdloanFundByParaId(key: ParaId) {
+  const bestNumber = useBestNumber();
+
+  return useApiQuery(['parachain_crowdloan_funds', key], async (api) => {
+    if (bestNumber) {
```
Comment (silva-fj) [inline src/api/hooks/useFunds.ts]: Should this query be `enabled` only when `bestNumber` is defined?

File: src/api/hooks/useFunds.ts [hunk 2]
```diff
@@ -18,39 +19,57 @@ export const CROWD_PREFIX = stringToU8a('modlpy/cfund');
...
-
-      const funds = rawFunds ? optFundMulti.transform(paraIds, rawFunds) : [];
-      const leases = optLeaseMulti.transform(paraIds, rawLeases ?? []);
-
-      const minContribution = api?.consts.crowdloan?.minContribution as BlockNumber;
-
-      if (bestNumber && funds && leases) {
-        return createResult(bestNumber, minContribution, funds, leases);
+      if (bestNumber) {
+        return await getFunds(paraIds, bestNumber, api);
```
Comment (silva-fj) [inline src/api/hooks/useFunds.ts:32]: is it important to get new data when `bestNumber` changes? or the data does not change that often?

**Review by silva-fj — DISMISSED** [review]

File: src/api/hooks/useFunds.ts (hunk 2 above)
Comment (ahoseinian) [reply src/api/hooks/useFunds.ts:32]: I haven't seen it change at all on polkadot website. I don't think it's really important.

File: src/api/hooks/useFunds.ts (hunk 1 above)
Comment (ahoseinian) [reply src/api/hooks/useFunds.ts]: yes, added ✅

**Review by silva-fj — APPROVED** [review]

**Pull request merged**